ast.go

```diff
@@ -28,13 +28,23 @@ import (
 type Serializer interface {
 	// Serialize writes the statement/expression to the io.Writer. If an
 	// error is returned the io.Writer may contain partial output.
-	Serialize(w io.Writer) error
+	Serialize(w io.Writer, interpolate bool) error
```

Comment:
Yeah, I didn't fully think this through. I believe I was thinking  you'd pass `sqlWriter` to `Serialize`. That doesn't seem horrible, though. The generality of being able to pass an `io.Writer` isn't being used (AFAIR).